src/Blorc.OpenIdConnect.Tests/Services/AccessTokenExpirationDelegatingHandlerFacts.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace Blorc.OpenIdConnect.Tests.Services
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net.Http;
+    using System.Text;
+    using System.Text.Json;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Moq;
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class AccessTokenExpirationDelegatingHandlerFacts
+    {
+        [TestFixture]
+        public class The_SendAsync_Method
+        {
+            [Test]
+            public async Task Calls_SignOutRedirectAsync_When_The_Token_Is_Expired_Async()
```

Comment:
This is a *great* unit test. Also make sure to add one when the token is not expired so we never introduce a "signout issue" with non-expired tokens.

Reply:
I don't think it's necessary. 

We currently have two handlers: one for appending the token and another for checking its expiration. This separation of concerns ensures clear distinctions in responsibilities and prevents any unintended behavior.

To add the handlers we have to do this 

    builder.Services
        .AddHttpClient<WeatherForecastHttpClient>(client => client.BaseAddress = new Uri(baseUrl))
        .AddAccessToken()
        .AddAccessTokenExpiration();

Will also update the readme.md

Reply:
But if a token is *not* expired, it should not sign out. That part is not covered by tests.

Reply:
My mistake, sorry